deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -180,6 +193,7 @@ rules:
       - pods
       - services
       - configmaps
+      # Needed for cloudbeat
```

Comment:
Either keep Needed or Required to have all lines aligned

Reply:
done https://github.com/elastic/elastic-agent/pull/708/commits/df21c4c24e4dfcb919b88da82eb5261e001e23bf